fvm/evm/debug/tracer.go

```diff
@@ -93,26 +99,20 @@ func (t *CallTracer) Collect(txID gethCommon.Hash) {
 			}
 		}()
 
-		res, err := t.tracer.GetResult()
-		if err != nil {
-			l.Error().Err(err).Msg("failed to produce trace results")
-		}
-
-		// reset tracing to have fresh state
-		if err := t.ResetTracer(); err != nil {
-			l.Error().Err(err).
-				Str("traces", string(res)).
-				Msg("failed to reset trace")
+		res, found := t.resultsByTxID[txID]
+		if !found {
+			l.Error().Str("txID", txID.Hex()).
+				Msg("trace result not found")
```

Comment:
```suggestion
			l.Error().Msg("trace result not found")
```

txID is included in the `l`

Reply:
fixed, thanks